model/src/main/scala/org/broadinstitute/dsde/rawls/StringUtils.scala

```diff
@@ -0,0 +1,25 @@
+package org.broadinstitute.dsde.rawls
+
+import org.broadinstitute.dsde.rawls.model.{Attributable, AttributeName, ErrorReport, ErrorReportSource}
+import spray.http.StatusCodes
+
+trait StringUtils {
```

Comment:
Open to arguments but I think I prefer the trait way.  Agree with you on the name.